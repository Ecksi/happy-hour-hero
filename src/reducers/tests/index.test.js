import { createStore } from 'redux';
import { rootReducer } from '../index';

const store = createStore(rootReducer);

describe('rootReducer', () => {
  it('matches the snapshot', () => {
    expect(rootReducer).toMatchSnapshot();
  });

  it('drinkSpecials should match the default state given an empty action', () => {
    expect(store.getState().drinkSpecials).toEqual([]);
  });

  it('foodSpecials should match the default state given an empty action', () => {
    expect(store.getState().foodSpecials).toEqual([]);
  });

  it('filteredRestaurants should match the default state given an empty action', () => {
    expect(store.getState().filteredRestaurants).toEqual([]);
  });

  it('happyHours should match the default state given an empty action', () => {
    expect(store.getState().happyHours).toEqual([]);
  });

  it('location should match the default state given an empty action', () => {
    expect(store.getState().location).toEqual({ "address": null, "latitude": 39.750801, "longitude": -104.996595 });
  });

  it('restaurants should match the default state given an empty action', () => {
    expect(store.getState().restaurants).toEqual([]);
  });
});