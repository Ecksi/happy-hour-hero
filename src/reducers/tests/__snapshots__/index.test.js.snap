// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer matches the snapshot 1`] = `[Function]`;
